projects.go

```diff
@@ -35,6 +35,17 @@ type ProjectsService struct {
 	client *Client
 }
 
+// BasicProject included in other service responses (such as todos).
+type BasicProject struct {
+	ID                int        `json:"id"`
+	Description       string     `json:"description"`
+	Name              string     `json:"name"`
+	NameWithNamespace string     `json:"name_with_namespace"`
+	Path              string     `json:"path"`
+	PathWithNamespace string     `json:"path_with_namespace"`
+	CreatedAt         *time.Time `json:"created_at"`
+}
```

Comment:
Please place this below the actual `Project` struct.

Reply:
I was follow `BasicUser` and `User` placements. Anyway, done.